### PR TITLE
Bugfix/609 printer bulk functions broken

### PR DESCRIPTION
**Pull request by NotExpectedYet**
Absolute goof

### CODE REVIEW

**Comment by davidzwa** [comment]
Pls add a test or inform me if you want me to do it :) you know me, seeing is believing

**Comment by NotExpectedYet** [comment]
@davidzwa fixed

File: server_src/routes/printers.js [hunk 1]
```diff
@@ -127,8 +127,15 @@ router.get("/groups", ensureAuthenticated, async (req, res) => {
 });
 router.post("/printerInfo", ensureAuthenticated, async (req, res) => {
   const id = req.body.i;
-  const returnPrinter = await PrinterClean.getPrintersInformationById(id);
-  res.send(returnPrinter);
+  let returnedPrinterInformation;
+  if (!id) {
+    returnedPrinterInformation = PrinterClean.listPrintersInformation();
+  } else {
+    returnedPrinterInformation = await PrinterClean.getPrintersInformationById(
```
Comment (davidzwa) [inline server_src/routes/printers.js:134]: Doesnt need await

**Review by davidzwa — APPROVED** [review]
With the tests I think it looks good

**Pull request merged**